docs/uninstalling.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Uninstalling Dep
+---
+
+## Uninstalling
+
+To uninstall `dep` itself, follow these instructions, depending on how you installed `dep` originally.
+
+### If you installed `dep` by executing the `install.sh` script via curl
+
+If you installed `dep` by `curl`'ing the `install.sh` script, it is safe to simply delete the installed binary file.
```

Comment:
suggestion: replace `curling the install.sh script` with "If you installed using the install.sh script" there is more than one way to get that script and the command ticks make it a bit hard to read 😀

Reply:
good point. will change!